### PR TITLE
Build OIDC verifier

### PR DESCRIPTION
**Pull request by dekiel**
**Description**

Changes proposed in this pull request:

- prowjobs for building oidc-token-verifier image.

See: https://github.com/orgs/kyma-project/projects/22/views/70?pane=issue&itemId=56193547

### CODE REVIEW

**Comment by dekiel** [comment]
This PR can be merged once oidc-token-verifier will be merged in to main branch.

**Review by Sawthis — APPROVED** [review]

**Pull request merged**
